doc/index.md

```diff
@@ -0,0 +1,71 @@
+# docstub user guide
+
+**Version**:
```

Comment:
I don't know how to insert a variable in here...

Reply:
For now, we could hard-write it.

Reply:
Honestly, I think we can skip this for now. It should be pretty clear that the user guide reflects the current version of the repo. Longterm this should end up in a website with a version switcher anyway. :)